playwright/src/main/java/com/microsoft/playwright/impl/Serialization.java

```diff
@@ -207,6 +219,17 @@ private static <T> T deserialize(SerializedValue value, Map<Integer, Object> idT
       return (T) value.b;
     if (value.s != null)
       return (T) value.s;
+    if (value.u != null) {
+      try {
+        return (T)(new URL(value.u));
+      } catch (MalformedURLException e) {
+        return null;
```

Comment:
Don't swallow the exception, `throw new PlaywrightException("Unexpected value: " + value.u, e);`